<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
background-color: rgb(58, 58, 58);</string>
  </property>
  <widget class="QLineEdit" name="tb_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dot">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(208, 208, 0);</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_equal">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(208, 208, 0);</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(208, 208, 0);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sub">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(208, 208, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_multi">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(208, 208, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_div">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(208, 208, 0);</string>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mod">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ma">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
